docs/how-xtrabackup-works.md

```diff
@@ -1,123 +1,149 @@
+4. Copy the log files up to the specified positions without the need to lock the resources for an extended period.
+5. Use the information from the `STORAGE_ENGINES` section to handle engine-specific log data, if applicable.
+
+It's important to note that the `log_status` table does not have indexes, and you cannot do operations like `TRUNCATE TABLE`.
+
+
+## Prepare phase
+
+The `prepare` phase uses the MySQL crash recovery process. This automated process uses transaction logs to return the database to a consistent state. The process works in the following way:
+
+* MySQL checks the InnoDB transaction log files to determine the database state
+
+* MySQL applies any uncommitted changes from the transaction logs to the database
+
+Percona XtraBackup performs crash recovery in the prepare phase before restoring the database. 
+
+The backed-up MyISAM and InnoDB tables are eventually consistent because, after the prepare (recovery) process, InnoDB’s data is rolled forward to the point at which the backup is completed, not rolled back to where it started. This point matches where the lock was taken, so the MyISAM data and the prepared InnoDB data are in sync.
+
+In brief, though, the `xtrabackup` binary enables you to do operations such as streaming and incremental backups with various combinations of copying the data files, copying the log files, and applying the logs to the data.
 
 ## Restoring a backup
 
-To restore a backup with xtrabackup you can use the [--copy-back] or
-[--move-back] options.
+To restore a backup with `xtrabackup`, you can use the [--copy-back] or [--move-back] options.
+
+The `xtrabackup` binary checks if the following directories exist in the `my.cnf` configuration file and then reads from the following locations:
+
+* datadir
+
+* innodb_data_home_dir
+
+* innodb_data_file_path
+
+* innodb_log_group_home_dir
+
+
+### copy-back option
+
+When you are restoring a database using XtraBackup, you would use the `--copy-back` option in the following scenario:
+
+* You have previously taken a backup of your database using XtraBackup.
+
+* You want to restore the database from the backup files.
+
+* Instead of manually copying the backup files to the appropriate locations, you can use the `--copy-back` option.
+
+This option automatically copies the MyISAM tables, indexes, InnoDB tables, indexes, and log files from the backup location to their respective locations in the MySQL data directory and preserves the file attributes during the copying process. Note that after using the `--copy-back` option, you may need to change the ownership of the copied files to the `mysql` user before starting the database server.
+
+The restore happens in the following order:
+
+* Copies the MyISAM tables, indexes, and the .MRG, .MYD, .MYI, .CSM, .CSV, `.sdi`, and `par` files 
+
+* Copies the InnoDB tables and indexes 
+
+* Copies the log files
```

Comment:
PXB stopped creating the redo log files. It will let server recreate redo files.